budcamp/Homework4/Homework4/IPhone.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Homework4
+{
+    internal interface IPhone
```

Comment:
wtf is this? why this is an interface?

Reply:
cause it was the theme of lecture... Must it be a class?

Reply:
there was a slide "Interface vs abstract class", or have you forgot about it?